six/modules/c++/six.sicd/source/ComplexXMLParser10x.cpp

```diff
@@ -181,6 +181,21 @@ XMLElem ComplexXMLParser10x::convertImageFormationToXML(
     createDouble("MinProc", imageFormation->txFrequencyProcMin, txFreqXML);
     createDouble("MaxProc", imageFormation->txFrequencyProcMax, txFreqXML);
 
+    XMLElem segmentElem = NULL;
```

Comment:
Similarly here, use the C++ structure to check for existence and fix exception message